frontend/src/styles/Login.module.css

```diff
@@ -0,0 +1,111 @@
+.joinOurCommunity {
+    font-family: Work Sans;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 3.2vh;
+    line-height: 3.2vh;
+
+    margin-top: 10vh;
+
+    text-align: center;
+    letter-spacing: 0.1em;
+
+    color: #000000;
+}
+
+.loginToCreate {
+    justify-content: center;
+
+    font-family: Work Sans;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 2.3vh;
+    line-height: 2.3vh;
+
+    text-align: center;
+
+    color: #000000;
+}
+
+.textInput {
+    height: 4.7vh;
+    width: 17%;
+
+    background: #E9E9E9;
+    border: none;
+    border-radius: 5px;
+
+    margin-bottom: 1.4vh;
+
+    /* input text */
+
+    padding-left: 1.3%;
+    font-family: Lato;
+    font-style: normal;
```

Comment:
can you remove the extra newlines/comments/styling that doesn't do anything like `font-style: normal` in all the css